src/bootstrap/src/utils/change_tracker.rs

```diff
@@ -446,4 +446,9 @@ pub const CONFIG_CHANGE_HISTORY: &[ChangeInfo] = &[
         severity: ChangeSeverity::Info,
         summary: "Added new option `build.tidy-extra-checks` to specify a default value for the --extra-checks cli flag.",
     },
+    ChangeInfo {
+        change_id: 143048,
+        severity: ChangeSeverity::Warning,
+        summary: "The default check stage has been changed to 1. It is no longer possible to `x check` with stage 0. All check commands have to be on stage 1+.",
```

Comment:
Discussion: maybe also mention cross-compile check of (rustc) must also be at stage 2?

Reply:
> Cross-compile check of (rustc) must also be at stage 2

Well, that's not strictly true. We now auto-bump and when you ask for stage 1 cross-compile check, bootstrap will silently do a stage 2 check.

Reply:
Ugh, right.